src/pyhf/infer/intervals.py

```diff
@@ -15,7 +15,7 @@ def _interp(x, xp, fp):
     return tb.astensor(np.interp(x, xp.tolist(), fp.tolist()))
 
 
-def upperlimit(data, model, scan, level=0.05, return_results=False):
+def upperlimit(data, model, scan, level=0.05, return_results=False, **hypotest_kwargs):
```

Comment:
please add more information in the docstring on this additional kwarg (e.g. provide a link to the hypotest function for information on what can be passed in)